### PR TITLE
DOC-11078: Improves description of functions throughout

### PR DESCRIPTION
**Pull request by franrob-projects**
Improves descriptions of: 

- scope1
- collection1
- collection2
- Index_replicas

Superseds -> https://github.com/couchbase/docs-sync-gateway/pull/742/commits/c1690f5bd5459ea0293578f9a3440535ff8684dc

### CODE REVIEW

File: modules/ROOT/pages/scopes-and-collections-config.adoc [hunk 1]
```diff
@@ -34,7 +34,8 @@ Gain maximum flexibility in organizing and accessing your data by customizing it
 Create a custom collection with the default scope can lead to efficient data organization and quicker access control management.
 
 * *default scope/default collection* +
-Work within the default Scopes and Collections. This makes managing data easy and simple without creating extra scopes or collections. 
+Work within the default Scopes and Collections. 
+This makes managing data easy and simple without creating extra scopes or collections. 
 +
```
Comment (osfameron) [inline modules/ROOT/pages/scopes-and-collections-config.adoc:39]: trailing plus?

**Review by bfavini — APPROVED** [review]

File: modules/ROOT/pages/scopes-and-collections-config.adoc [hunk 2]
```diff
@@ -161,6 +162,7 @@ A Collection can contain a set of documents.
 You have the option to use the default provided by Couchbase or customize it to fit your specific needs.
 
 === Collection2
+Collection2 is a mirror of collection1, and is storage for documents that are labelled as "default". Collectio2's import filter ensures that only documents labelled as "mobile" are imported. Any changes made to documents in collection2 are also made to collection1, and vice versa.
```
Comment (osfameron) [inline modules/ROOT/pages/scopes-and-collections-config.adoc:165]: New line (x2)
"Collectio2" typo

File: modules/ROOT/pages/scopes-and-collections-config.adoc [hunk 3]
```diff
@@ -215,4 +217,5 @@ For more information about others, see xref:import-filter.adoc[Import Filters].
 }
 ----
 
-According to this configuration example, the database shouldn't have any index replicas created.
+The num_index_replicas property specifies the number of replicas that should be created for each index in the database.
```
Comment (osfameron) [inline modules/ROOT/pages/scopes-and-collections-config.adoc:220]: Put technical things like `num_index_replicas` in backticks to format as code

File: modules/ROOT/pages/scopes-and-collections-config.adoc [hunk 4]
```diff
@@ -103,16 +104,16 @@ A Scope can contain a set of Collections.
 You have the option to use the default provided by Couchbase or customize it to fit your specific needs.
 
 === scope1
-
-The "scopes" field is not required for upgrading to Sync Gateway 3.1. 
-Even if this field is not specified in the configuration, the previous versions of the configuration file are still compatible with the new version.
-Therefore, the provided examples are not essential for upgrading to Sync Gateway's latest version.
+There are two collections within scope1; collection1 and collection2.
```
Comment (osfameron) [inline modules/ROOT/pages/scopes-and-collections-config.adoc:107]: I'd mark e.g. `scope1` etc. also within backticks to format as code

**Pull request merged**